lib/v2/douban/router.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function (router) {
+    router.get('/book/latest', require('./other/latest_book'));
+    router.get('/book/rank/:type?', require('./book/rank'));
+    router.get('/bookstore', require('./other/bookstore'));
+    router.get('/celebrity/:id/:sort?', require('./other/celebrity.js'));
+    router.get('/channel/:id/:nav?', require('./channel/topic.js'));
+    router.get('/channel/:id/subject/:nav', require('./channel/subject.js'));
```

Comment:
```suggestion
    router.get('/channel/:id/subject/:nav', require('./channel/subject.js'));
    router.get('/channel/:id/:nav?', require('./channel/topic.js'));
```
This two need to be swapped